.github/workflows/test-conventions.yml

```diff
@@ -0,0 +1,41 @@
+name: test/conventions
+
+on:
+  push:
+    branches:
+        - feature/*
+        - release/**
+        - 'v[4-9]+.[0-9]+'
+
+  pull_request:
+    branches:
+        - feature/*
+        - release/**
+        - 'v[4-9]+.[0-9]+'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+          os: [ ubuntu-latest, windows-latest, macos-13 ]
```

Comment:
do we want to test those on all OS kinds or win is enought? @ppekrol

Reply:
windows is enough